utils/syscall.c

```diff
@@ -382,9 +382,342 @@ static const myst_syscall_pair_t _pairs[] = {
     PAIR(SYS_myst_interrupt_thread),
     PAIR(SYS_myst_pre_launch_hook),
     /* add new entries here! */
+    /* NOTE: when you add a new syscall here, add it to its corresponding group
+       below as well! */
     {0, NULL},
 };
 
+const myst_syscall_group_t _groups[] = {
+    {"desc",
+     100,
```

Comment:
There seems to be 98 items, excluding the 2 commented out

Reply:
Oh no thank you for pointing that out, that would be a bug!